src/UserChip/UserChip.spec.tsx

```diff
@@ -1,57 +1,62 @@
 import testImage from  '../../assets/user.png';
-import TestUtil from '../Util/TestUtil';
 
 import UserChip from './UserChip';
+import { render, screen } from '@testing-library/react';
+import * as React from 'react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
 
 describe('<UserChip />', () => {
 
-  let wrapper;
-
-  beforeEach(() => {
-    wrapper = TestUtil.mountComponent(UserChip);
-  });
-
   it('is defined', () => {
     expect(UserChip).not.toBeUndefined();
   });
 
   it('can be rendered', () => {
-    expect(wrapper).not.toBeUndefined();
+    const { container } = render(<UserChip />);
+    expect(container).toBeVisible();
   });
 
   it('determines initials from given user name', () => {
-    wrapper.setProps({userName: 'Shinji Kagawa'});
-    const test = wrapper.instance().getInitials();
-    expect(test).toBe('SK');
+    render(<UserChip userName="Shinji Kagawa" />);
+    const chip = screen.getByText('SK').parentElement;
```

Comment:
```suggestion
    const chip = screen.getByText('SK');
```
wouldn't that be enough?

Reply:
in this case, yes